docs/source/_src_user_guide/state_propagation/environment_setup/create_bodies/available.rst

```diff
@@ -0,0 +1,129 @@
+* A thrust acceleration in Tudat is calculated from two models: (1) an engine model, which defined the body-fixed direction of the thrust, and the magnitude of the thrust (2) the orientation of the body in space, defined by its rotation model
+* For a non-spherical central body, the current orientation of this central body has an indirect influence on the altitude at which a vehicle with a given *inertial* state is located
+
+Two rotation models, which are typically used for vehicles under thrust, and/or vehicles in an atmosphere, are the following:
+
+* The rotation model :func:`~tudatpy.numerical_simulation.environment_setup.rotation_model.aerodynamic_angle_based`, which calculates the body's rotation based on the angle of attack, sideslip angle and bank angle. Note that these angles are definend w.r.t. the relative wind. This model is typical when using, for instance, a re-entry simulation. It imposes these three angles, and calculates the body orientation by combination with the latitude, longitude, heading angle, flight path angles. There is a related model, :func:`~tudatpy.numerical_simulation.environment_setup.rotation_model.zero_pitch_moment_aerodynamic_angle_based`, that uses the same setup, but does not impose the angle of attack, but caculates by imposing aerodynamic pitch trim (zero pitch moment).
+* The rotation model :func:`~tudatpy.numerical_simulation.environment_setup.rotation_model.custom_inertial_direction_based`, which is typical when calculating dynamics of a vehicle under thrust. It is based on linking a body-fixed  direction (now limited to the body-fixed x-axis) to an arbitrary inertial direction. This allows the thrust (assuming that this is aligned with this same body-fixed direction) to be guided in an inertial direction determined by a user-defined model. 
+
+Ephemeris models
+----------------
+
+An ephemeris is arguably the most fundamental of the environment models: it defines *where* a body is located in space. 
+
+Use of Spice
+~~~~~~~~~~~~
+
+For many typical applications, natural body ephemerides will be calculated from Spice kernels. In some cases, a user may find that the default Spice kernels are insufficient for their purposes, due to one of two reasons:
+
+* The body for which the state is required *is* in the ephemeris Spice kernel, but the time at which the state is needed lies outside of the bounds for which the Spice kernel has data
+* The body for which the state is required *is not* in the ephemeris Spice kernel
+
+In both cases, a user should load additional Spice kernels. This can be done using the :func:`~tudatpy.interface.spice.load_kernel`. Spice kernels for many bodies may be found in a number of places. The 'goto' place for Spice kernels for ephemerides is the NAIF website (developers of Spice), which you can find `here <https://naif.jpl.nasa.gov/pub/naif/generic_kernels/spk/>`_.
+
```

Comment:
Top-level comment: I am not sure whether this is the best place to mention this (actually I'm quite sure that it is not), but I think that it might be quite useful to add a page or section somewhere stating explicitely what the spice kernels which are loaded by default contain (e.g. ephemerides available from which bodies, etc.). I know that there are also arguments against providing such a list, and that it will requires some updates, but it would address a recurring question from students and it would clarify which body settings are available per default. What are your thoughts about this?